### PR TITLE
Yessenia Rodriguez node-notes-cli

### PR DESCRIPTION
**Pull request by yesseniarod**
Screen Shot 2021-03-11 at 5 13 50 PM

### CODE REVIEW

File: node-notes-cli/app.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+for (const key in dataModel.notes) {
+  if (process.argv[2] === 'delete' && key === process.argv[3]) {
+    delete dataModel.notes[key];
+    const dataObject = JSON.stringify(dataModel, null, 2);
+    fs.writeFile('data.json', dataObject, 'utf8', err => {
+      if (err) throw err;
+    });
+  }
+}
```
Comment (thebearingedge) [inline node-notes-cli/app.js:28]: You can access the note directly using bracket notation without the loop.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: node-notes-cli/app.js (hunk 1 above)
Comment (yesseniarod) [reply node-notes-cli/app.js]: Screen Shot 2021-03-12 at 5 47 12 PM

File: node-notes-cli/app.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  const key = dataModel.nextId;
+  dataModel.notes[key] = process.argv[3];
+  dataModel.nextId++;
+  const dataObject = JSON.stringify(dataModel, null, 2);
+  fs.writeFile('data.json', dataObject, 'utf8', err => {
+    if (err) throw err;
+  });
+}
+
+// for (const key in dataModel.notes) {
```
Comment (thebearingedge) [inline node-notes-cli/app.js:20]: Please do not turn in commented out code 😱

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: node-notes-cli/app.js (hunk 2 above)
Comment (yesseniarod) [reply node-notes-cli/app.js]: I'm sorry 😓

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**